core/arch/arm/mm/core_mmu.c

```diff
@@ -101,6 +101,10 @@ static struct memaccess_area nsec_shared[] = {
 	MEMACCESS_AREA(CFG_SHMEM_START, CFG_SHMEM_SIZE),
 };
 
+#ifdef CFG_TEE_SDP_MEM_BASE
```

Comment:
Feels like we should have another name here describing the feature instead of just grabbing a "random" define used when working with SDP? What about `CFG_WITH_SDP`?